tests/integration/assessment/test_workflows.py

```diff
@@ -4,21 +4,46 @@
 from databricks.sdk.retries import retried
 from databricks.sdk.service.iam import PermissionLevel
 
+from databricks.labs.ucx.hive_metastore import TablesCrawler
+
 
 @retried(on=[NotFound, InvalidParameterValue], timeout=timedelta(minutes=8))
-def test_running_real_assessment_job(ws, installation_ctx, make_cluster_policy, make_cluster_policy_permissions):
+def test_running_real_assessment_job(
+    installation_ctx,
+    make_cluster_policy,
+    make_cluster_policy_permissions,
+    make_schema,
+    make_table,
+    sql_backend,
+    inventory_schema,
+):
     ws_group_a, _ = installation_ctx.make_ucx_group()
     cluster_policy = make_cluster_policy()
     make_cluster_policy_permissions(
         object_id=cluster_policy.policy_id,
         permission_level=PermissionLevel.CAN_USE,
         group_name=ws_group_a.display_name,
     )
+
+    source_schema = make_schema(catalog_name="hive_metastore")
+    managed_table = make_table(schema_name=source_schema.name)
+    external_table = make_table(schema_name=source_schema.name, external=True)
+    tmp_table = make_table(schema_name=source_schema.name, ctas="SELECT 2+2 AS four")
+    view = make_table(schema_name=source_schema.name, ctas="SELECT 2+2 AS four", view=True)
+    non_delta = make_table(schema_name=source_schema.name, non_delta=True)
```

Comment:
```suggestion
    source_schema = installation_ctx.make_schema(catalog_name="hive_metastore")
    managed_table = installation_ctx.make_table(schema_name=source_schema.name)
    external_table = installation_ctx.make_table(schema_name=source_schema.name, external=True)
    tmp_table = installation_ctx.make_table(schema_name=source_schema.name, ctas="SELECT 2+2 AS four")
    view = installation_ctx.make_table(schema_name=source_schema.name, ctas="SELECT 2+2 AS four", view=True)
    non_delta = installation_ctx.make_table(schema_name=source_schema.name, non_delta=True)
```

see tests.integration.conftest.MockRuntimeContext.make_schema and related